providers/logingov.go

```diff
@@ -103,10 +103,12 @@ func checkNonce(idToken string, p *LoginGovProvider) (err error) {
 			return nil, myerr
 		}
 		body, myerr := ioutil.ReadAll(resp.Body)
-		resp.Body.Close()
 		if myerr != nil {
 			return nil, myerr
 		}
+		if myerr = resp.Body.Close(); myerr != nil {
```

Comment:
Should this have been migrated to the request builder?

Reply:
Yes I think it should be, I didn't look for `http.Get`, only `http.NewRequest` and `http.NewRequestWithContext`, can you move it to the request builder pattern?